### PR TITLE
Upgrade to webpack 2.0 and deps

### PR DESCRIPTION
**Pull request by frederickfogerty**

### CODE REVIEW

File: src/components/IndexColumn.jsx [hunk 1]
```diff
@@ -100,7 +109,10 @@ class IndexColumn extends React.Component {
         onMouseLeave={ this.handleMouseLeave }>
 
         <div style={ [styles.outer, styles.suiteTree] }>
-          <SuiteTree ref="suiteTree" selectedSuite={ currentSuite } width={ width } />
+          <SuiteTree
+            ref={ c => { this.suiteTree = c; } }
```
Comment (philcockfield) [inline src/components/IndexColumn.jsx]: Clever.

File: src/server/webpack-config.js [hunk 2]
```diff
@@ -26,14 +26,16 @@ const productionEnvPlugin = new webpack.DefinePlugin({
...
     exclude: /(node_modules|bower_components)/,
   };
 
   // Add optional plugins.
   if (isRelayEnabled) {
-    loader.loaders[0] += `?plugins[]=${ fsPath.join(__dirname, '../relay/babel-relay-plugin') }`;
+    loader.loaders[0].query = {
+      plugins: fsPath.join(__dirname, '../relay/babel-relay-plugin'),
+    };
   }
```
Comment (philcockfield) [inline src/server/webpack-config.js]: Not for this PR - but I'm thinking of dropping Relay support.  Thoughts?

**Comment by philcockfield** [comment]
Cool @frederickfogerty .  And tree-shaking just comes along for free with the 2.0 version of webpack?  Didn't see any specific configuration settings, or was that the 

      new webpack.LoaderOptionsPlugin({ minimize: true })

?

**Comment by philcockfield** [comment]
👍 LGTM.

**Comment by frederickfogerty** [comment]
Before
image


After
image

**Pull request closed without merging**